OSDOCS-17172-bootstrap-cert CQA (#114912)
openshift/openshift-docs: security/certificate_types_descriptions/bootstrap-certificates.adoc

toc::[]

[role="_abstract"]

> Understand bootstrap certificates that enable kubelet transport layer security (TLS) bootstrapping when nodes join a cluster, including how the certificates are issued and rotated and how the certificates are managed.

Reviewer: This ^ isn't clear to me. Is this accurate/better?
```suggestion
You should understand how bootstrap certificates enable kubelet transport layer security (TLS) bootstrapping when nodes join a cluster, including how the certificates are issued and rotated and how the certificates are managed.
```

Author: Done

> You should understand how bootstrap certificates that enable kubelet transport layer security (TLS) bootstrapping when nodes join a cluster, including how the certificates are issued and rotated and how the certificates are managed.

Reviewer: @wgabor0427 One last tweak. 

```suggestion
You should understand how bootstrap certificates enable kubelet transport layer security (TLS) bootstrapping when nodes join a cluster, including how the certificates are issued and rotated and how the certificates are managed.
```

Author: Done

[id="bootstrap-certificates-purpose_{context}"]
== Purpose

The kubelet, in {product-title} 4 and later, uses the bootstrap certificate located in `/etc/kubernetes/kubeconfig` to initially bootstrap. This is followed by the bootstrap initialization process and the authorization of the kubelet to create a certificate signing request (CSR).

> In that process, the kubelet generates a CSR while communicating over the bootstrap channel. The controller manager signs the CSR, resulting in a certificate that the kubelet manages. For more information, see "Bootstrap initialization" and "Authorize kubelet to create a CSR" in the _Additional resources_ below.

Reviewer: Per ISG: [Do not use (below) to indicate a relative location in a document, as in “the information below”](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b). 

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b
```suggestion
In that process, the kubelet generates a CSR while communicating over the bootstrap channel. The controller manager signs the CSR, resulting in a certificate that the kubelet manages. For more information, see "Bootstrap initialization" and "Authorize kubelet to create a CSR" in the _Additional resources_ section.
```

Author: Done
